dashboard/config/blocks/Poetry/Poetry_startBehavior.json

```diff
@@ -18,55 +18,55 @@
         "options": [
           [
             "fluttering",
-            "fluttering"
+            "\"fluttering\""
```

Comment:
making sure i understand -- are the missing `\"` the reason behavior names were all coming through as undefined?